### DISCUSSION TITLE
Envelope Scale Factor

### DISCUSSION THREAD

**Discussion by NoiseTheorem** [SynthstromAudible/DelugeFirmware#793] [Ideas]
For many sound design tasks, slow attack and release times are necessary.  Unfortunately, even at the maximum value, the EGs are still a bit too fast for this purpose. 
I would propose adding a 'scale factor' parameter, that would allow for creating long eg's (x2, x3, x4, etc).

**Comment by m-m-adams** [comment]
Same as the other one - this is easy but needs a UI
